service/src/main/resources/application.properties

```diff
@@ -28,4 +28,7 @@ event-repository.enabled: false
 event-repository.url: http://<host>:<port>/<context-path-if-available>
 
 # lenientValidationEnabledToUsers true will perform the validation only on mandatory fields, non-mandatory validation failures add into Eiffel message as property remremGenerateFailures
-lenientValidationEnabledToUsers: false
\ No newline at end of file
+lenientValidationEnabledToUsers: false
+
+#Maximum size of events in array is fetched from REMReM property and checked during publishing.
```

Comment:
Maximum number of templates in array passed to /xxx endpoint. Please, replace xxx by real endpoint name.

Reply:
Done